res/sections/26-lec3pt2.tex

```diff
@@ -45,164 +45,169 @@ \section{Esempi di tecniche di Geolocalizzazione}
-in piedi).
-\item allenare il motore inferenziale con i dati collezionati
-\item applicare il motore inferenziale all'applicazione target.
-\end{itemize}
+Si chiama inferenza il processo di mappatura dei dati di basso livello a eventi significativi di alto livello.
+
+La pipeline di inferenza tipicamente comprende tre fasi: sensing, feature extraction e classification.
+
+La classificazione richiede un motore inferenziale o modello, che viene costruito in tre fasi:
+\begin{enumerate}
+\item Collezione dei dati grezzi con associata la corretta classificazione. Il dataset così formato dovrebbe comprendere anche situazioni che non stiamo cercando di individuare, ma che sono simili a quelle d'interesse (ad esempio se vogliamo individuare la camminata, dovremmo disporre di dati per la corsa e per lo stare in piedi).
+\item Addestramento del motore inferenziale (calcolo del modello) con i dati collezionati.
+\item Utilizzo del motore inferenziale nell'applicazione target per classificare i nuovi dati grezzi.
+\end{enumerate}
 
 \section{Feature Extraction}
+
 Questa fase prevede di identificare le features in un dataset che possono essere
 usate per inferire un particolare tipo di attività.
+
 L'insieme delle features selezionate dipende dal tipo di sensori e di attività
-che si deve individuare. Il processo di design tipicamente include anche analisi
+che si deve individuare. 
+
+Il processo di design tipicamente include anche analisi
 offline del training set per identificare le features corrette per il
-particolare motore inferenziale in genere questo tipo di analisi è fatta
-mediante un processo iterativo, in cui vengono testate features diverse.
+particolare motore inferenziale. 
+In genere questo tipo di analisi è fatta mediante un processo iterativo, in cui vengono testate features diverse.
+
 Un possibile esempio è quello dell'individuazione della conversazione: si può
-applicare FFT (trasformata di Fourier) ai samples audio, e comparare i training
-data che sono etichettati come "conversazione" e come ``rumore non
-conversazione''. Si seleziona come features la media e la standard deviation
-sulla potenza dell'FFT. A questo punto anche una semplice linea come treshold
-può fornire risultati relativamente accurati (anche se con un alto numero di
-falsi positivi).
-Nel caso di attività fisica ci si può basare sui dati dell'accelerometro,
-confrontando l'essere seduti, con l'essere in piedi, il camminare e il correre:
-le features possono essere la media, la deviazione standard e il numero di
-picchi.
+applicare FFT (trasformata di Fourier) ai samples audio e comparare i training
+data che sono etichettati come ``conversazione'' con quelli ``rumore non
+conversazione''. 
+Come features si possono utilizzare la media e lo scarto quadratico medio della potenza dell'FFT. 
+A questo punto anche una semplice linea come treshold può fornire risultati relativamente accurati (anche se con un alto numero di falsi positivi).
```

Comment:
Okok. Allora metti a posto sta robetta e tira su la linea 80 se hai voglia e poi buttiamo in dev